models/maverick/ethereum/maverick_v1_ethereum_trades.sql

```diff
@@ -21,29 +21,30 @@ WITH dexs AS
     SELECT
         t.evt_block_time AS block_time
         ,t.recipient AS taker
-        ,'' AS maker
+        ,CAST(NULL AS VARBINARY)as maker
         ,amountOut AS token_bought_amount_raw
         ,amountIn AS token_sold_amount_raw
-        ,CAST(NULL AS DOUBLE) AS amount_usd
+        ,NULL AS amount_usd
         ,CASE WHEN tokenAIn THEN f.tokenB ELSE f.tokenA END AS token_bought_address
         ,CASE WHEN tokenAIn THEN f.tokenA ELSE f.tokenB END AS token_sold_address
-        ,CAST(t.contract_address as string) as project_contract_address
+        ,t.contract_address as project_contract_address
         ,t.evt_tx_hash AS tx_hash
-        ,'' AS trace_address
+        ,CAST(NULL as array<bigint>) AS trace_address
```

Comment:
We started removing `trace_address` from models in dex_trades's upstream lineage, so this isn't needed.

Reply:
Ah sorry missed that one, fix it now.